docs/configuration/oauth.md

```diff
@@ -0,0 +1,72 @@
+# OAuth settings
```

Comment:
Not a blocker, but let's include information (sooner or later) about where to get these values from.